test/promql/tests/test_degraded_argoCD.yaml

```diff
@@ -0,0 +1,37 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+
+      # no argoCD app is degraded
+      - series: 'argocd_app_info{health_status="Degraded", name="first-test-app", dest_namespace="not-degraded"}'
+        values: '0x15'
+
+      # argoCD has a degraded app
+      - series: 'argocd_app_info{health_status="Degraded", name="second-test-app", dest_namespace="has-degraded"}'
+        values: '1x15'
```

Comment:
It'd be nice to either have a more complex series here or some additional tests to cover patterns like flapping.

Reply:
Done, I added another test case for a flappy app